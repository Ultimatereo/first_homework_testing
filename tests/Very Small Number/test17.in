5
3 3 3 1 3 2
1 0 5 1 2 4
5 0 5 0 4 3
3 0 5 2 1 1
0 0 2 4 1 5
