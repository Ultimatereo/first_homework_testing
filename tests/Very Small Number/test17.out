-0.967105
-0.539474
0.125
0.736842
1.80263
